src/main/scala/com/atomist/rug/runtime/Handler.scala

```diff
@@ -0,0 +1,19 @@
+package com.atomist.rug.runtime
+
+import com.atomist.param.{Parameter, Tag}
+
+/**
+  * Common stuff for Handlers
+  */
+trait Handler {
+
```

Comment:
It might add some consistency if Handler had a handle(handleable: T) method and everything that could be handled was a Handleable.
Since ParameterizedHandler is separate, there may need to be a ParameterizedHandleable that is similar but has parameters.